x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/api/crawler/create_crawler_index_api_logic.ts

```diff
@@ -15,23 +16,31 @@ export interface CreateCrawlerIndexArgs {
 }
 
 interface CreateCrawlerIndexRequest {
-  index_name: string;
   language: LanguageForOptimization;
 }
 
 export interface CreateCrawlerIndexResponse {
-  created: string; // the name of the newly created index
+  updated: string; // the name of the updated index
```

Comment:
question: Why is this `updated`? We're only ever creating a new index here, right?